addon/components/frost-select.js

```diff
@@ -343,14 +343,14 @@ export default Component.extend({
 
       switch (state.lastAction) {
         case 'SELECT_HOVER':
+          break
```

Comment:
This code is meant to be triggered in the case that either of these actions were triggered